app/controllers/kyc/organisation-url/get.controller.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { response } = require('../../../utils/response')
+const { isSwitchingCredentialsRoute, isAdditionalKycDataRoute, getCurrentCredential } = require('../../../utils/credentials')
+
+module.exports = (req, res, next) => {
+  const isSwitchingCredentials = isSwitchingCredentialsRoute(req)
+  const collectingAdditionalKycData = isAdditionalKycDataRoute(req)
+  const currentCredential = getCurrentCredential(req.account)
+
+  if (collectingAdditionalKycData && !req.account.requires_additional_kyc_data) {
+    return next(new Error('requires_additional_kyc_data flag is false on request'))
```

Comment:
```suggestion
    return next(new Error('requires_additional_kyc_data flag is not enabled for gateway account'))
```

Reply:
Thanks - have updated the file and test.